src/AuxDisplay.jsx

```diff
@@ -43,12 +46,12 @@ export default class AuxDisplay extends React.Component {
         }
     }
     play() {
-        if (this.el) {
+        if (this.el && this.el.paused) {
```

Comment:
There is no issue there, it's just redundant. This function gets called multiple times per second on the aux video, so I didn't want to bother the video element that often with spurious `play()` calls.